src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const first = (a) => {
```

Comment:
it is not necessary to give the function some name. We can just make it anonymous and return right away
```suggestion
  return (a) => {
```